### PR TITLE
Dash Cytoscape updates

### PR DESCRIPTION
**Pull request by ohaanika**
Relevant Issues: #551, #550 

Post-merge checklist:

The master branch is auto-deployed to `dash.plot.ly`.
Once you have merged your PR, wait 5-10 minutes and check dash.plot.ly 
to verify that your changes have been made.

- [x] I understand

### CODE REVIEW

**Comment by alexcjohnson** [comment]
Nice! Great job debugging the callback problems.

I wonder if we can take it a step further though: Now that `dcc.Markdown` automatically handles syntax highlighting and dedenting, can we get rid of `PythonSnippet` entirely? Replace:
```py
    PythonSnippet('''
    @app.callback(Output('cytoscape-callbacks-1', 'layout'),
                  [Input('dropdown-callbacks-1', 'value')])
        def update_layout(layout):
            return {'name': layout}
    '''),
```

With:
````py
    dcc.Markdown('''
    ```py
    @app.callback(Output('cytoscape-callbacks-1', 'layout'),
                  [Input('dropdown-callbacks-1', 'value')])
        def update_layout(layout):
            return {'name': layout}
    ```
    ''')
````

Which to my mind anyway is much clearer because you don't have to wonder what's going on in `PythonSnippet`.

And then it turns out a lot of the previous `PythonSnippet` components are preceded or followed by other `dcc.Markdown` components so they can be merged (after removing the remaining `dedent` calls) - see https://github.com/plotly/dash-docs/pull/549 for an example.

**Comment by ohaanika** [comment]
@alexcjohnson Apologies for taking long to get back to this PR! 

Since I see that `dcc.Markdown()` is used every where else in the docs, I did as you suggested to keep things consistent. To be specific:

1. I had to rewrite `PythonSnippet('''...''')` as `dcc.Markdown(dedent(''' ```py ... ``` '''), style=styles.code_container)` to maintain the styling that have we for code snippets (i.e. grey left border). 
2. I did not merge `dcc.Markdown` components, as the `dcc.Markdown` component capturing code snippets had different styling which had to specified (as mentioned above). 

Come to think of it... I just remembered with Cytoscape R docs, I remember using the equivalent of `Display` instead of the equivalent of `PythonSnippet` for such code snippets, since it happened by chance that the code snippets that we want to show wouldn't output anything to begin with. Just a side note!

File: tutorial/cytoscape/applications_chapter.py [hunk 1]
```diff
@@ -37,10 +36,12 @@
     
     ''')),
 
-    PythonSnippet('''
+    dcc.Markdown(dedent('''
```
Comment (alexcjohnson) [inline tutorial/cytoscape/applications_chapter.py]: `dedent` is no longer necessary as of Dash 1.0 https://github.com/plotly/dash-core-components/pull/569

Comment (ohaanika) [reply tutorial/cytoscape/applications_chapter.py]: Oh is this change yet to be implemented in the rest of the docs too?
e.g. https://github.com/plotly/dash-docs/blob/master/tutorial/table/sizing_chapter.py

Comment (alexcjohnson) [reply tutorial/cytoscape/applications_chapter.py]: right, we haven't stripped it out everywhere yet, that's going to take a while... but we can at least avoid adding new uses.

Comment (ohaanika) [reply tutorial/cytoscape/applications_chapter.py]: In that case I'll remove ALL `dedent()` for Cytoscape docs (i.e. including non-code snippets).

Comment (ohaanika) [reply tutorial/cytoscape/applications_chapter.py]: Removed!

Comment (alexcjohnson) [reply tutorial/cytoscape/applications_chapter.py]: Great! now if `dedent` isn't used anymore in these files, can you :hocho: `from textwrap import dedent`?

Comment (ohaanika) [reply tutorial/cytoscape/applications_chapter.py]: Good point, just committed 👍

**Pull request merged**